lib/Constants/src/Constants.h

```diff
@@ -10,40 +10,58 @@ const float PROPORTIONAL_GAIN = 0.01;
 // Per car constants
 #if model == 21
 const int MODEL_NUMBER = model;
-// Pins
-const int EG_INTERRUPT_PIN = 37;
-const int WL_INTERRUPT_PIN = 36;
-const int ESTOP_IN_PIN = 33;
-const int ESTOP_OUT_PIN = 32;
-const int LED_1_PIN = 28;
-const int LED_2_PIN = 29;
 
+// Pins
+namespace pin
+{
+    const int EG_INTERRUPT_PIN = 37;
```

Comment:
For these variable names, could they be changed to not end with _PIN?
They will already be called with pin::LED_1 so the pin at the end is redundant